src/ffi/dial_ffi.rs

```diff
@@ -28,13 +28,13 @@ use anyhow::Result;
 
 use crate::proxy::grpc_proxy::GRPCProxy;
 
-/// The Ffi interface, returned as a pointer by init_rust_runtime. User should keep this pointer until freeing the runtime.
-pub struct Ffi {
+/// The DialFfi interface, returned as a pointer by init_rust_runtime. User should keep this pointer until freeing the runtime.
+pub struct DialFfi {
```

Comment:
@njooma will python sdk need to make any changes when this is renamed?

Reply:
oh no! I can change it back if we need

Reply:
Nope I believe this should be ok!